custom/icds_reports/management/commands/sql_scripts/district_data_pull_2.sql

```diff
@@ -0,0 +1,35 @@
+COPY(SELECT
+    awc.district_name,
+    awc.state_name,
+    agg.state_id,
+    agg.district_id,
+    CASE WHEN agg.month='{month_1}' THEN agg.cf_initiation_in_month ELSE 0 END as cf_initiation_in_month_{column_1},
```

Comment:
you will need to sum it with group by district. as this table has multiple rows per location.